dependabot/generate-build-time-stats.py

```diff
@@ -0,0 +1,181 @@
+import os
+import sys
+import time
+
+from github import Github, InputGitAuthor, GithubException
+import github
+
+import constants
+
+SLEEP_INTERVAL = 30 # 30s
+MAX_WAIT_CYCLES = 180 # script timeout is 90 minutes
+
+ballerina_bot_username = os.environ[constants.ENV_BALLERINA_BOT_USERNAME]
+ballerina_bot_token = os.environ[constants.ENV_BALLERINA_BOT_TOKEN]
+ballerina_bot_email = os.environ[constants.ENV_BALLERINA_BOT_EMAIL]
+ballerina_reviewer_bot_token = os.environ[constants.ENV_BALLERINA_REVIEWER_BOT_TOKEN]
+
+ballerina_org_name="ballerina-platform"
+github = Github(ballerina_bot_token)
+
+def main():
+
+    github = Github(ballerina_bot_token)
```

Comment:
we can remove this instance since it is created earlier

Reply:
Done